documentation/reference.md

```diff
@@ -168,7 +168,7 @@ If you have additional data to report that does not fit any of the columns provi
 |Recipient Org:Identifier|A globally unique identifier for this organisation. This is important to enable data on funders and recipients to be linked up across different grant-makers. The [Organisation Identifier Standard](http://www.threesixtygiving.org/standard/identifiers/#toc-organisation-identifier) guidance explains how to create this ID, based either on the known company or charity number, or upon identifiers held in the grant-maker's internal systems.|string|True|
 |Recipient Org:Name|Organisation name|string|True|
 |Recipient Org:Department|The department or sub-unit of this organisation making or receiving the grant.|string|False|
-|Recipient Org:Contact Name|-|string|False|
+|Recipient Org:Contact Name|The person at this organisation to contact about making or receiving the grant.|string|False|
```

Comment:
Because of the way these documents are generated I think this sentence is problematic (it applies in a few places and trying to cover grant making/giving is difficult)

I'd suggest simplifying e.g.:
"A named person at this organisation" "The name of a contact person at this organisation" or something.

You may want to contact them about something other than the grant itself - maybe about the data, or about the organisation itself.